### ISSUE TITLE
unify (publish) API key with "girder" API key  

### ISSUE THREAD

**Issue by yarikoptic** [dandi/dandi-publish#135]
ATM client uses girder API key to be able to upload data.

"publish" functionality (see https://github.com/dandi/dandi-cli/issues/164) requires another django key.

There should be a single key provided to users to be able to interact with the archive.

Feel free to close if duplicate or move if better aligns with `dandiarchive`.

**Comment by mgrauer** [comment]
Yes, eventually there will be a single key to interact with the archive, once the API is unified, for all mutation operations.

Until the API is unified, end users will have no need to interact with the Django side via an API key because all of their interactions will be read only.

**Comment by yarikoptic** [comment]
yes -- in the long run, for users there should be a single key.  This issue is just to make sure that it would be the case.

But the whole issue was triggered by our current need for developers (of `dandi-cli`) to be able to get or set (for a docker-compose run) a KEY for django so we could unit-test downloading from released dandisets etc
